sources/packages/backend/apps/queue-consumers/src/processors/schedulers/institution-integration/ier12-integration/_tests_/e2e/ier12-factory.ts

```diff
@@ -0,0 +1,342 @@
+import {
+  Application,
+  DisbursementSchedule,
+  DisbursementScheduleStatus,
+  DisbursementValue,
+  FullTimeAssessment,
+  InstitutionLocation,
+  ProgramYear,
+  Student,
+  StudentAssessment,
+} from "@sims/sims-db";
+import {
+  E2EDataSources,
+  createFakeInstitutionLocation,
+  createFakeStudent,
+  createFakeUser,
+  ensureProgramYearExists,
+  saveFakeApplicationDisbursements,
+  saveFakeStudent,
+} from "@sims/test-utils";
+import { createFakeSINValidation } from "@sims/test-utils/factories/sin-validation";
+import {
+  IER12Application,
+  IER12Assessment,
+  IER12Disbursement,
+  IER12Offering,
+  IER12Program,
+  IER12Student,
+  IER12TestInputData,
+} from "./models/data-inputs";
+import { addDays, dateDifference, getISODateOnlyString } from "@sims/utilities";
+import * as faker from "faker";
+
+/**
+ * Save all IER12 related records providing all data that
+ * will be part of the IER12 file generation.
+ * @param db data source helper.
+ * @param testInputData msfaa test input data.
```

Comment:
`msfaa` ?? shouldn't it be `IER12 test input data`?

Reply:
Changed.